openapi/doc.js

```diff
@@ -178,6 +178,12 @@ Privacy Policy: <https://www.inaturalist.org/privacy>`
         name: "Authorization",
         in: "header",
         description: "User or application JSON Web Token (application tokens only available to official apps)"
+      },
+      appAndUserJwtRequired: {
+        type: "apiKey",
+        name: "Authorization",
+        in: "header",
+        description: "User and application JSON Web Token (application tokens only available to official apps)"
```

Comment:
We should specify that they need to both be in the same header field, separated by a comma.

Reply:
I updated the description to specify they both must be present in a comma-separated list